esp/esp/program/modules/handlers/finaidapprovemodule.py

```diff
@@ -59,62 +59,51 @@ class Meta:
     def finaidapprove(self, request, tl, one, two, module, extra, prog):
         context = {}
         message = ""
-        num_approved = 0
-        num_errors = 0
+        users_approved = []
+        users_error = []
 
         # The following code was copied and modified from finaid_approve.py in useful_scripts
 
-        reqs = FinancialAidRequest.objects.filter(program__name=prog)
+        reqs = FinancialAidRequest.objects.filter(program=prog)
 
         # computing len will use the same query we're probably going to use later and
         # populate the query set cache (whereas if we used .exists() or .count(), they
         # wouldn't, and the later iteration would hit the database again)
         if len(reqs) == 0:
             message = "No requests found."
-            context["message"] = message
+            context["error"] = message
             return render_to_response(self.baseDir()+'finaid.html', request, context)
 
         if request.method == 'POST':
             context['POST'] = True
 
             # ITERATE & APPROVE REQUESTS
             userchecklist = request.POST.getlist("user")
-
-            message += "{} New Approval{} ({} error{}): "
+            approve_blanks = request.POST.get('approve_blanks', False);
 
             def is_blank(x):
-                return x is None #or re.match(r'^(\s)*$', x)
+                return x is None or x == ""
```

Comment:
I think you can catch both of these cases with `bool(x)`, so you might not even need a separate function.

Reply:
I tried it out and ``bool(x)`` does work, although I'm thinking the separate function makes it easier to tell what the program is doing:
``if is_blank(req.household_income) or is_blank(req.extra_explaination):``
vs.
``if bool(req.household_income) and bool(req.extra_explaination):``
Although maybe not.. What do you think?

Reply:
I agree it's clearer with the function, and it's probably just as efficient time-wise.